package.json

```diff
@@ -11,6 +11,7 @@
     "cookie": "^0.1.2",
     "ejs": "^2.3.1",
     "extend": "^2.0.1",
+    "json2csv": "^3.3.0",
```

Comment:
@trevren11 I looked into it. Looks like the COPY method in postgres can only be used if you have the [pg-copy-streams module](https://github.com/brianc/node-pg-copy-streams). At this point since it would just be changing one module to another is it worth making the change?